guides/user/pipeline-expressions/index.md

```diff
@@ -24,33 +24,24 @@ I can now set up my pipeline to take a `command` parameter, when I manually exec
 
 ![](2.png)
 
-When the pipeline runs, it will substitute the value I have entered for `command` with the value `sleep 40000` that I have entered. 
+When the pipeline runs, it resolves the expression to what I’ve entered for `command`.
 
 ## Where can I use Pipeline Expressions in Spinnaker?
 
-![](3.png)
+> **Warning**
+>
+> Sometimes adding an expression stops your ability to use the UI. For example, if you use a parameter for the account field in a cluster deployment Stage, you will see a spinning loader from the UI when you try to edit this cluster.
 
 You can usually type in a value in the text fields of the Pipeline stages in Spinnaker. 
 
+[comment]: <> ![](3.png)
+<img width="600" src="3.png" alt="alt_text"> 
```

Comment:
Should this be `alt_text`? or something more descriptive?

Reply:
Something more descriptive ;) 
I put it there to test the image b/c I was having trouble with the markdown syntax for the image.